manifests/i/Insco/FaForever/0.0.1.23/Insco.FaForever.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Insco.FaForever
+PackageVersion: 0.0.1.23
+PackageLocale: en-US
+Publisher: Qingdao INSCO Network Technology Co., Ltd.
```

```suggestion
Publisher: kjxbyz
```
This needs to match control panel exactly

---

What is the control panel?